presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,434 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+database. The database contains definitions of all resource groups and the rules to select them.
+Unlike the file-based resource manager, changes to the configuration in the database take effect
+immediately and do not require a restart of the Presto server.
+
+Both methods have their pros and cons. File-based resource management is simpler to set up but less flexible,
+while database-based resource management is more complex to set up but offers more flexibility and dynamic changes.
+
+File Resource Group Manager
+---------------------------
+
+The File Resource Group Manager in PrestoDB is a way to manage resources using a JSON configuration file.
+This file contains definitions of all resource groups and rules for selecting the appropriate resource group
+for a given query.
+
+To set up a File-based Resource Group Manager:
+
+1. Create the file ``etc/resource-groups.properties``
+2. Set the ``resource-groups.configuration-manager`` property to ``file``.
+3. Create a JSON file with the definitions of your resource groups and rules.
+4. Specify the location of this file in the Presto server's configuration file
+   under the property ``resource-groups.config-file``.
+5. To complete the setup, follow the steps in File Resource Group Manager.
+
+Here's a brief guide on how to set it up:
+
+1. Create a JSON file: This file should contain the definitions for your resource groups.
+Each resource group can specify things like the maximum memory, the maximum queued queries,
+and the maximum running queries.
+
+2. Specify the file in the Presto configuration: In the ``etc`` directory of your Presto installation,
+create a file named ``resource-groups.properties``. In this file, add a line that specifies the location of
+your JSON file: ``resource-groups.config-file = <file_path>``, where ``<file_path>`` is the path to your JSON file.
+
+3. Restart the Presto server: After creating the JSON file and pointing to it in the Presto configuration,
+restart your Presto server. The new resource groups will take effect after the restart.
+
+.. code-block:: text
 
     resource-groups.configuration-manager=file
-    resource-groups.config-file=etc/resource_groups.json
+    resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Presto data directory.
+Database Resource Group Manager
+-------------------------------
+The Database Resource Group Manager in PrestoDB is a way to manage resources using a relational database.
 
-Resource Group Properties
--------------------------
+To set up a Database-based Resource Group Manager:
 
-* ``name`` (required): name of the group. May be a template (see below).
+1. Create the file ``etc/resource-groups.properties``
+2. Set the ``resource-groups.configuration-manager`` property to ``db``.
+3. Set up a relational database that Presto can access.
```

Comment:
Similar to my previous comment at line 69, I suggest you combine these steps into a single set of steps.